components/build/build-service/kustomization.yaml

```diff
@@ -16,12 +16,9 @@ patchesJSON6902:
       value: build-service
 
 images:
-- name: controller
+- name: quay.io/redhat-appstudio/build-service
```

Comment:
The name is `quay.io/redhat-appstudio/build-operator`

Reply:
Fixed reference on build-service side and updated commit reference